bodhi/server/scripts/manage_releases.py

```diff
@@ -37,7 +37,11 @@
 
 @click.group()
 def main():
-    """Create and manage releases in Bodhi."""
+    """
+    Create and manage releases in Bodhi.
+
+    (This utility has been deprecated. Please use 'bodhi releases' instead.)
+    """
     pass  # pragma: no cover
```

Comment:
Let's do the deprecation warning here instead of in each command:

```
click.echo("This utility has been deprecated. Please use 'bodhi releases' instead.)
```